cmake/caches/cn/fast/mlir-cuda-80.cmake

```diff
@@ -19,10 +19,10 @@ set(CMAKE_C_COMPILER_LAUNCHER ccache CACHE STRING "")
 set(CMAKE_CXX_COMPILER_LAUNCHER ccache CACHE STRING "")
 set(CMAKE_CUDA_COMPILER_LAUNCHER ccache CACHE STRING "")
 set(CMAKE_INTERPROCEDURAL_OPTIMIZATION OFF CACHE BOOL "")
-set(CMAKE_EXE_LINKER_FLAGS_INIT "-fuse-ld=lld" CACHE STRING "")
-set(CMAKE_MODULE_LINKER_FLAGS_INIT "-fuse-ld=lld" CACHE STRING "")
-set(CMAKE_SHARED_LINKER_FLAGS_INIT "-fuse-ld=lld" CACHE STRING "")
+set(CMAKE_EXE_LINKER_FLAGS_INIT "-fuse-ld=bfd" CACHE STRING "")
+set(CMAKE_MODULE_LINKER_FLAGS_INIT "-fuse-ld=bfd" CACHE STRING "")
+set(CMAKE_SHARED_LINKER_FLAGS_INIT "-fuse-ld=bfd" CACHE STRING "")
```

Comment:
is bfd faster than lld?

Reply:
In the building container the lld is somehow useless, bfs is the workaround method